src/main/scala/mesosphere/marathon/Main.scala

```diff
@@ -52,6 +54,14 @@ class MarathonApp extends App {
       new HttpModule(conf) {
         // burst browser cache for assets
         protected override val resourceCacheControlHeader = Some("max-age=0, must-revalidate")
+
+        override def provideHandlerCollection(instrumentedHandler: InstrumentedHandler,
```

Comment:
Is this related or something extra?

Does it provide metrics or something similar? Or is it a performance patch? Please document or make the intention clear by using other means.